R/snapshot.R

```diff
@@ -13,8 +13,12 @@
 #'   defaults to \code{packages.csv}. If it is NULL,
 #'   to output file is created and the result is returned
 #'   as a data frame.
+#' @param lib.loc character vector describing the 
+#'   location of R library trees to search through, or 
+#'   \code{NULL} for all known trees (see 
+#'   \code{\link[base]{.libPaths}}).
 #' @param recommended if TRUE then recommended packages
-#' will be included in the snapshot.
+#'   will be included in the snapshot.
```

Comment:
This is nitpicking, but please try not to change the whitespace. Obviously, this single line is fine, but excessive amounts of ws changes make it harder to follow what is new. Thanks!